common/lib/xmodule/xmodule/tests/__init__.py

```diff
@@ -59,6 +61,8 @@ def __init__(self, **kwargs):
         kwargs.setdefault('id_reader', id_manager)
         kwargs.setdefault('id_generator', id_manager)
         kwargs.setdefault('services', {}).setdefault('field-data', DictFieldData({}))
+        self.tracker = DjangoTracker()
+        tracker.register_tracker(self.tracker)
```

Comment:
@mulby @dino-cikatic - the same pattern is used here - lms/djangoapps/django_comment_client/base/tests.py (around line 1007 and in a few other tests). Seems like we should be okay with this model for now? Perhaps we can add a cleanup task to our backlog.